wbooks/src/App/Auth/api.js

```diff
@@ -0,0 +1,4 @@
+export const API = {
+  baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1',
+  endpoint: '/users/sessions'
+}
```

Comment:
You can use axios instances and set the base url to it.
So you can move the instance to `src/config/api.js` and import your `api.js` everywhere.